packages/editor-ui/src/dialogs/edit-image.tsx

```diff
@@ -162,10 +163,11 @@ const imagePanel =
   
     {alt !== undefined
       ?  <Field label={t("Alternative text")}>
-          <Input 
-            value={alt} 
+          <Textarea
+            value={alt}
             onChange={(_ev, data) => setAlt(data.value)}
             placeholder={t("(Optional)")}
+            style={{ height: "100px" }}
```

Comment:
Could we also set `resize = {true}`?

Reply:
Yes, manual resize is possible.

Reply:
https://github.com/quarto-dev/quarto/pull/644/commits/615dc1ee4726a7be91400729cf55b7f10fcb4479